pkgs/top-level/all-packages.nix

```diff
@@ -71,6 +71,25 @@ with pkgs;
 
   common-updater-scripts = callPackage ../common-updater/scripts.nix { };
 
+  ### Push NixOS tests inside the fixed point
+
+  nixosTests =
+    let
+      system = builtins.currentSystem;
```

Comment:
This doesn’t work, I think. cc @Ericson2314

Reply:
I don't really know: NixOS test derivations are supposed to be built on the local system, which I'd guess to be `builtins.currentSystem`? Not sure at all though, so cc @Ericson2314 indeed :) I.must find it hard to imagine what's the cross-derivation story of NIxOS tests.

Reply:
@Ericson2314 ping :) (2 months sounds like a reasonable waiting time :p)